x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/endpoint_activity_log.tsx

```diff
@@ -79,7 +79,7 @@ export const EndpointActivityLog = memo(
 
     return (
       <>
-        <EuiFlexGroup direction="column" style={{ height: '85vh' }}>
+        <EuiFlexGroup direction="column" style={{ height: '85vh' }} responsive={false}>
```

Comment:
I know this change is not from this pr but could you create a StyledComponent with this specific height? I think this is how the EUI team wants us to modify component styles instead of by the style prop directly.

Reply:
This is actually my change as well. Forgot to clean this up in an earlier PR. 😅

Reply:
done 8d361962661d7031a8720324554d003b6ee3731c